expenses-manager-frontend/src/main/frontend/src/app/right-sidebar/right-sidebar.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="horizontal-container"> 
```

Comment:
De asemenea acest bar ar trebui sa fie pana in capat. Pe ecranele mai mari se termina brusc la un moment dat:

![image](https://github.com/user-attachments/assets/0ec8c739-e0a0-4d99-857a-100e60869418)

Reply:
Poti sa maresti bara cu Day, Week..iar right-sidebar sa ramana la dimensiunea curenta.